src/main/java/com/example/fizzbuzz/controller/FizzBuzzDBController.java

```diff
@@ -1,40 +1,47 @@
 package com.example.fizzbuzz.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
-
-import static com.example.fizzbuzz.fizzbuzzServices.FizzBuzzDBService.connectionAndExecute;
-import static com.example.fizzbuzz.fizzbuzzServices.FizzBuzzResultService.setResult;
+import com.example.fizzbuzz.fizzbuzzServices.FizzBuzzResultService;
+import com.example.fizzbuzz.model.FizzBuzzDB;
+import com.example.fizzbuzz.repository.FizzBuzzRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 public class FizzBuzzDBController {
-    
-    @GetMapping("/insertValue")
-    public static void addValues(@RequestParam(value = "id") Long id, @RequestParam (value = "myNumber") Integer myNumber, @RequestParam (value = "name") String name) {
-        String query = "INSERT INTO fizz_buzzdb (id, my_number, name, result) VALUES ( '" +id+"' , '" +myNumber+"' , '"+name+"', '"+ setResult(myNumber)+"')";
-        connectionAndExecute(query);
-        System.out.println("Value Inserted Successfully!");
+
+    @Autowired
+    private FizzBuzzRepository fizzBuzzRepository;
+
+    @PostMapping("/insertValue")
+    public @ResponseBody String addNewFizzBuzz (@RequestParam(value = "id") Long id, @RequestParam (value = "myNumber") Integer myNumber, @RequestParam (value = "name") String name) {
+        FizzBuzzDB entry = new FizzBuzzDB();
+        entry.setId(id);
+        entry.setName(name);
+        entry.setMyNumber(myNumber);
+        entry.setResult(FizzBuzzResultService.setResult(myNumber));
+        fizzBuzzRepository.save(entry);
+        return "Saved";
```

Comment:
We can improve responses from the controller by just returing the entity in case it was created/updated (the `save` method should return it)

Reply:
Done